packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -10,10 +10,12 @@ import { useComboboxContent } from "../../hooks/useComboboxContent";
 import { useComboboxAccessibility } from "../../hooks/useComboboxAccessibility";
 import { ComboboxContentProps } from "../../Combobox.types";
 import { COMBOBOX_MENU_ID } from "../../constants";
+import { ComboboxContext } from "../../ComboboxProvider";
 
 export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
   const optionsExist = props.options.length > 0;
   const { optionsListRef } = useComboboxContent(props.open, props.selected);
+  const { onClear, onSelectAll } = useContext(ComboboxContext);
```

Comment:
so `useComboboxContent` is already a hook made specifically for the `ComboboxContent.tsx` to clean up and group the things it cares about.

inside there you'll see that we're already doing `const { shouldScroll } = useContext(ComboboxContext);`

so let's avoid having duplicate calls to the same Context.

instead, we can do

```
const { onClear, onSelectAll, shouldScroll } = useContext(ComboboxContext);
....
  return {
    onClear,
    onSelectAll,
    optionsListRef,
  };
```

then in ComboboxContent 

`const { onClear, onSelectAll, optionsListRef } = useComboboxContent(...)`